.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -16,42 +24,34 @@ jobs:
+        CONFIG: linux_aarch64_c_compiler_version10cuda_compiler_version11.2cxx_compiler_version10
         UPLOAD_PACKAGES: 'True'
-        DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cuda:11.2
-      linux_aarch64_cuda_compiler_version11.0:
-        CONFIG: linux_aarch64_cuda_compiler_version11.0
+        DOCKER_IMAGE: quay.io/condaforge/linux-anvil-aarch64-cuda:11.2
+      linux_aarch64_c_compiler_version9cuda_compiler_version11.0cxx_compiler_version9:
+        CONFIG: linux_aarch64_c_compiler_version9cuda_compiler_version11.0cxx_compiler_version9
         UPLOAD_PACKAGES: 'True'
         DOCKER_IMAGE: quay.io/condaforge/linux-anvil-aarch64-cuda:11.0
-      linux_aarch64_cuda_compiler_version11.1:
-        CONFIG: linux_aarch64_cuda_compiler_version11.1
+      linux_aarch64_c_compiler_version9cuda_compiler_version11.1cxx_compiler_version9:
+        CONFIG: linux_aarch64_c_compiler_version9cuda_compiler_version11.1cxx_compiler_version9
         UPLOAD_PACKAGES: 'True'
         DOCKER_IMAGE: quay.io/condaforge/linux-anvil-aarch64-cuda:11.1
-      linux_aarch64_cuda_compiler_version11.2:
-        CONFIG: linux_aarch64_cuda_compiler_version11.2
+      linux_ppc64le_c_compiler_version10cuda_compiler_version11.2cxx_compiler_version10:
+        CONFIG: linux_ppc64le_c_compiler_version10cuda_compiler_version11.2cxx_compiler_version10
         UPLOAD_PACKAGES: 'True'
-        DOCKER_IMAGE: quay.io/condaforge/linux-anvil-aarch64-cuda:11.2
-      linux_ppc64le_cuda_compiler_version10.2:
-        CONFIG: linux_ppc64le_cuda_compiler_version10.2
+        DOCKER_IMAGE: quay.io/condaforge/linux-anvil-ppc64le-cuda:11.2
+      linux_ppc64le_c_compiler_version9cuda_compiler_version10.2cxx_compiler_version9:
+        CONFIG: linux_ppc64le_c_compiler_version9cuda_compiler_version10.2cxx_compiler_version9
         UPLOAD_PACKAGES: 'True'
         DOCKER_IMAGE: quay.io/condaforge/linux-anvil-ppc64le-cuda:10.2
-      linux_ppc64le_cuda_compiler_version11.0:
-        CONFIG: linux_ppc64le_cuda_compiler_version11.0
+      linux_ppc64le_c_compiler_version9cuda_compiler_version11.0cxx_compiler_version9:
+        CONFIG: linux_ppc64le_c_compiler_version9cuda_compiler_version11.0cxx_compiler_version9
         UPLOAD_PACKAGES: 'True'
         DOCKER_IMAGE: quay.io/condaforge/linux-anvil-ppc64le-cuda:11.0
-      linux_ppc64le_cuda_compiler_version11.1:
-        CONFIG: linux_ppc64le_cuda_compiler_version11.1
+      linux_ppc64le_c_compiler_version9cuda_compiler_version11.1cxx_compiler_version9:
+        CONFIG: linux_ppc64le_c_compiler_version9cuda_compiler_version11.1cxx_compiler_version9
         UPLOAD_PACKAGES: 'True'
         DOCKER_IMAGE: quay.io/condaforge/linux-anvil-ppc64le-cuda:11.1
-      linux_ppc64le_cuda_compiler_version11.2:
-        CONFIG: linux_ppc64le_cuda_compiler_version11.2
-        UPLOAD_PACKAGES: 'True'
-        DOCKER_IMAGE: quay.io/condaforge/linux-anvil-ppc64le-cuda:11.2
   timeoutInMinutes: 360
```

Comment:
Looks like this one got dropped. Maybe some other tweaks are need in `conda_build_config.yaml`?

Reply:
I have no clue... the CI file `.ci_support/linux_ppc64le_c_compiler_version10cuda_compiler_version11.2cxx_compiler_version10.yaml` is still there...

Reply:
Wait, all 4 ppc64le builds got run? From 10.2 to 11.2.